tests/gradients/core/test_pulse_gradient.py

```diff
@@ -846,7 +846,7 @@ def test_raises_for_invalid_reorder_fn(self, reorder_fn):
 
 
 @pytest.mark.jax
-@pytest.mark.parametrize("dev_name", ["default.qubit", "default.qubit.jax"])
+@pytest.mark.parametrize("dev_name", ["default.qubit"])
```

Comment:
Same comment about removing the parametrization and any test function variables conditionally set based on device name.